tests/driver_ccs811/main.c

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (C) 2018 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @brief       Test application for the Sensirion SHT30/SHT31/SHT35 device driver
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ * @file
+ *
+ * The test application demonstrates the use of the CCS811. It uses the default
+ * configuration parameters, that is, the measurement mode #ccs811_mode_1s with
+ * one measurement per second.
+ *
+ * The application can use both approaches to wait for new data
+ *
+ * - using the data-ready interrupt #ccs811_int_data_ready (USE_CSS811_DATA_READY_INT=1), or
+ * - using the data-ready status function #ccs811_data_ready (USE_CSS811_DATA_READY_INT=0).
+ */
+
+#define USE_CSS811_DATA_READY_INT  (0) /* 1 - use data-ready interrupt
```

Comment:
I will try

Reply:
Ok, works with
```
CFLAGS=-DUSE_CSS811_DATA_READY_INT make BOARD=esp8266-esp-12x -C tests/driver_ccs811
```
Since this produces a define in ```riotbuild.h``` like
```
#define USE_CSS811_DATA_READY_INT 1
```
it can be also used with ```#if```.